src/Microsoft.Health.Dicom.SchemaManager.Console/Properties/launchSettings.json

```diff
@@ -2,7 +2,10 @@
   "profiles": {
     "Microsoft.Health.Dicom.SchemaManager": {
       "commandName": "Project",
-      "commandLineArgs": "apply --connection-string \"server=(local);Initial Catalog=DICOM3;TrustServerCertificate=True;Integrated Security=True\" --version 20"
+      "commandLineArgs": "apply --connection-string \"server=(local);Initial Catalog=DICOM3;TrustServerCertificate=True;Integrated Security=True\" --version 20",
```

Comment:
I know this isn't your change, but our tests use `DICOM` as the initial catalog.